.github/ISSUE_TEMPLATE/issue-report.yaml

```
@@ -0,0 +1,42 @@
+name: "Issue report"
+description: Report documentation issues such as inaccuracies, broken links, typos, or missing information.
+title: "[Issue]: "
+labels: ["Issue"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi 👋. Thanks for taking the time to fill out this issue report!
+        This form will create an issue that Elastic's docs team will triage and prioritize.
+        You can also open a PR instead.
+  - type: dropdown
+    attributes:
+      label: Type of issue
+      description: What type of issue are you reporting?
+      multiple: false
+      options:
+        - Not clear
+        - Missing information
+        - Could be improved
+        - I can't find what I'm looking for
```

Should we have an "Other" section?

---

```suggestion
        - Not clear
        - Missing information
        - Could be improved
        - I can't find what I'm looking for
        - Other
```

---

Are these answers going to affect how we handle the issue? IMO if they're not going to help with routing we should narrow it to the smallest list possible (some of these currently seem overlapping to me e.g. "not clear" and "could be improved").

---

If the affect the relative urgency assigned to the issues, my vote is that "inaccurate" and "missing" would be more critical than "needs improvement"

---

Good point. Something like this?

```suggestion
        - Inaccurate
        - Missing information
        - I can't find what I'm looking for
        - Other
```